docs/source/commandline/commandline-all.rst

```diff
@@ -1328,9 +2210,14 @@ Core Video Options
 
     The default is ``1``.
 
-    This is supported with all video output types (``bgfx``, ``d3d``, etc) on
+    This is supported with all video output types ( ``bgfx``, ``d3d``, etc) on
```

Comment:
Does it really not work with `bgfx` on Linux/macOS?

Reply:
Good question. That text predates implementation on other platforms. I would ASSUME bgfx should work in all cases as it's designed to be completely cross-platform and all. If it DOESN'T work, that should be a bug report filed. I'll change the text accordingly.